CraftTweaker2-API/src/main/java/crafttweaker/api/util/Math.java

```diff
@@ -0,0 +1,169 @@
+package crafttweaker.api.util;
+
+import crafttweaker.annotations.ZenRegister;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.util.MathUtil")
+@ZenRegister
+public class Math {
```

Comment:
I think you misunderstood my previous review, I would like this to be:
```java
@ZenClass("crafttweaker.util.Math")
@ZenRegister
public class MathUtil {
```
instead, that way we don't pollute the `Math` name in Java.


It could alternatively be:
```java
@ZenClass("crafttweaker.util.MathUtil")
@ZenRegister
public class MathUtil {
```
Whatever you think makes more sense, as long as the actual Java class name is not `Math`

Reply:
Ah, yeah, sorry, I get it now